### PR TITLE
wooden walls

### PR DESCRIPTION
**Pull request by unid15**
also makes deconstructing walls grab sheets from pool

sprites from tg

### CODE REVIEW

File: code/game/turfs/simulated/walls.dm [hunk 1]
```diff
@@ -50,22 +50,19 @@
...
-			else
-				var/M = text2path("/obj/item/stack/sheet/mineral/[mineral]")
-				new M(src)
-				new M(src)
-				getFromPool(/obj/item/stack/sheet/metal, get_turf(src))
+			var/M = text2path("/obj/item/stack/sheet/mineral/[mineral]")
+			if(M)
+				getFromPool(M, get_turf(src), 2)
+
+		if(devastated)
```
Comment (ComicIronic) [inline code/game/turfs/simulated/walls.dm]: Walls shouldn't be dropping material sheets unless they aren't devastated

File: code/game/turfs/simulated/walls.dm [hunk 2]
```diff
@@ -50,22 +50,19 @@
...
 	else
-		if(!devastated)
-			new /obj/structure/girder(src)
-			if(mineral == "metal")
-				getFromPool(/obj/item/stack/sheet/metal, get_turf(src), 2)
-			else
-				var/M = text2path("/obj/item/stack/sheet/mineral/[mineral]")
-				new M(src)
-				new M(src)
+		if(mineral == "metal")
```
Comment (ComicIronic) [inline code/game/turfs/simulated/walls.dm]: This system is moronic and I don't like its extension.

It should be using material IDs and then getting sheet type from the material datum, not doing this weird path nonsense.

**Comment by Probe1** [comment]
jarringly inconsistent style/10

**Comment by FudgePucker** [comment]
Looks more like metal painted brown instead of wood.

**Comment by NigglyWiggly** [comment]
But Probe, I'm pretty sure that's the exact same as the other material walls, just recolored

**Comment by ComicIronic** [comment]
That style of wall is the same as the other mineral walls, yes.
so ugly

**Comment by dylanstrategie** [comment]
The sprites look fair for a starter, they can always be improved later

**Comment by Probe1** [comment]
Huh.  I never really noticed.  Well don't take me into account I'm off base.

**Comment by bustygoku** [comment]
Chocolate

**Comment by SkowronX** [comment]
brown walls. got it

**Pull request merged**